staging/src/k8s.io/apiserver/hack/update-codegen.sh

```diff
@@ -1,4 +1,3 @@
-
 #!/usr/bin/env bash
 
 # Copyright 2018 The Kubernetes Authors.
```

Comment:
> staging/src/k8s.io/apiserver/hack/update-codegen.sh 100644 → 100755

Nit, should set the file permission in the first commit, rather than update it in the third commit

Reply:
👍 I'll be squashing at the end